next-app/src/routes/projects/[project_code]/activities.ts

```diff
@@ -3,7 +3,7 @@ import { sf } from '$lib/fetch/server'
 export async function get({ params: { project_code }, request: { headers } }) {
 	const cookie = headers.get('cookie')
 
-	const activities = await get_activities({ project_code, cookie })
+	const activities = await get_activities({ project_code, cookie, start_date: undefined, end_date: undefined })
```

Comment:
I'd like to understand this one better, seems unnecessary to me but I'd like to hear your thoughts.

Reply:
Typescript says that the type `{ project_code, cookie }` isn't compatible with the type `{ project_code, cookie, start_date, end_date }` and flags this as a type error. I could have resolved this another way, such as making a type declaration that had `start_date` and `end_date` as optional, but that seemed overkill for a single usage.